generated/client/client.go

```diff
@@ -60,7 +60,7 @@ func JoinByFormat(data []string, format string) string {
 	}
 	return strings.Join(data, sep)
 }
-func (c Client) GetBooks(ctx context.Context, i *models.GetBooksInput) (models.GetBooksOutput, error) {
+func (c Client) GetBooks(ctx context.Context, i *models.GetBooksInput) (*[]models.Book, error) {
```

Comment:
yeah I would prefer either `[]models.Book` or `[]*models.Book` over the pointer to a slice